chef-universal-mingw32.gemspec

```diff
@@ -15,6 +15,7 @@ gemspec.add_dependency "win32-process", "~> 0.8.2"
 gemspec.add_dependency "win32-service", "~> 0.8.7"
 gemspec.add_dependency "windows-api", "~> 0.4.4"
 gemspec.add_dependency "wmi-lite", "~> 1.0"
+gemspec.add_dependency "win32-taskscheduler", "0.3.2"
```

Comment:
This should have a comparison rather than pin to a specific version, unless there is a specific reason we need only this version, in which case there needs to be a comment about why.

Reply:
Yes I will update that